steward/src/commands/scheduled_corks.rs

```diff
@@ -0,0 +1,13 @@
+mod shutdown;
+use shutdown::Shutdown;
+
+use abscissa_core::{clap::Parser, Command, Runnable};
+
+/// Schedule corks commands
+#[derive(Command, Debug, Parser, Runnable)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Schedule corks for target contract.\n This command schedules cork by calling a function from the target Cellar. This is a validator only \n command and can only be run by validators. It Schedules cork based on the height specified by the \n validators. Therefore, it'll execute the function when the chain reaches that height."
+)]
+pub enum ScheduledCorksCmd {
+    Shutdown(Shutdown),
```

Comment:
Also this command is called `SetShutdown` in the Aave cellar and has two parameters `shutdown: bool` and `exit_position: bool`:

https://github.com/PeggyJV/cellar-contracts/blob/b93b4393299aee35421f83ef4ca2689256e5b354/contracts/AaveV2StablecoinCellar.sol#L793

Reply:
Noted. You'll have to get that merged if that be the case